content/en/security/vulnerabilities/_index.md

```diff
@@ -16,48 +14,58 @@ further_reading:
 
-- View your container images on the [container images][2] page. 
-- Observe the number of vulnerabilities that exist in the container images (column name `VULNERABILITIES`).
+{{< img src="security/vulnerabilities/CSM_Vulnerabilities.png" alt="The CSM Vulnerability page sorting by unique vulnerabilities" width="100%">}}
 
-{{< img src="security/infrastructure_vulnerabilities/container_images_tab.png" alt="The Container Images tab highlighting vulnerabilities and container column sort feature" width="100%">}}
+## Manage container vulnerabilities
 
-View additional details about any vulnerability by clicking the container image and reviewing the **Vulnerabilities** tab:
+View your container images on the [container images][2] page. 
+Observe the number of vulnerabilities that exist in the container images (column name **vulnerabilities**). Additionally, sort by **source**, **image tag**, **repo digest**, and more.
 
-{{< img src="security/infrastructure_vulnerabilities/container_vulnerability.png" alt="Container Images overview page highlighting the vulnerabilities tab" width="100%">}}
+{{< img src="security/vulnerabilities/container_images_tab.png" alt="The Container Images tab highlighting vulnerabilities and container column sort feature" width="100%">}}
 
-## Identify impacted infrastructure
+View additional details about any vulnerability by clicking the container image and reviewing the **Vulnerabilities** tab.
 
-- Click any vulnerability to see more information. 
-- Click **View Vulnerability in CSM** to view all related infrastructure impacted by this vulnerability.
+Select a specific vulnerability to see its details, including which containers and hosts are affected, severity breakdown score, and recommended remediation steps. On the details explorer, you can also view impacted resources in CSM to gain better insights to your overall risk.
 
-This opens the CSM [Vulnerabilities][1] page which includes information about: 
+{{< img src="security/vulnerabilities/container_vulnerability_2.png" alt="Container Images overview page highlighting the vulnerabilities tab" width="100%">}}
 
-- A description of the vulnerability.
-- The date on which the vulnerability was last detected.
-- Recommended remediation steps.
-- Affected container images and hosts.
+Within CSM, the severity of a vulnerability is modified from the base score to take into account the business sensitivity of the environment where the vulnerability is detected. For example, if no production environment is detected, the severity is reduced.
 
-{{< img src="security/infrastructure_vulnerabilities/vulnerabilities_side_panel.png" alt="CSM Vulnerabilities side panel showing details of a vulnerability" width="100%">}}
+The Datadog Adjusted Vulnerability Score includes the full context of each impacted container or host:
+
+- The original vulnerability severity
+- Environment the infrastructure is running in
+- Known public exploit
 
 ## Review and remediate
```

Comment:
We have an **Explore vulnerabilities** and a **Review and remediate** section. In both cases, they're talking about the Vulnerabilities Explorer. I recommend combining the two sections and renaming it something like **Review and remediate vulnerabilities**.

Reply:
Yeah.. noted it's a bit duplicated, have moved some things around and removed redundant sections.